x-pack/test/security_solution_cypress/es_archives/risky_hosts/mappings.json

```diff
@@ -10,15 +10,9 @@
         "host": {
           "properties": {
             "name": {
-              "type": "text",
```

Comment:
'host.name' has to be a 'keyword'. 
https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#7-create-the-space-aware-ml_host_risk_score_latest_your-space-name-index-with-appropriate-mappings